linkyCrawler.py

```diff
@@ -2,32 +2,36 @@
 import sys
 import json
 import linkyClient
+import os
 import datetime
 from dateutil.relativedelta import relativedelta
 
+
 def main():
     """Main function"""
     parser = argparse.ArgumentParser()
     parser.add_argument('-u', '--username', required=True, help='Enedis username')
-    parser.add_argument('-p', '--password', required=True, help='Password')
+    parser.add_argument('-p', '--password', help='Enedis password. Can be read form LTI_LINKY_PASS Environment')
     parser.add_argument('-d', '--date', required=False, help='Day (dd/mm/aaaa)')
     args = parser.parse_args()
-
+    if args.password:
+        LTI_LINKY_PASS = args.password
+    else:
+        LTI_LINKY_PASS = os.getenv('LTI_LINKY_PASS')
```

Comment:
These lines replace that `lpass=${LTI_LINKY_PASS?"No Linky password specified"}`